returnn/tf/layers/basic.py

```diff
@@ -5728,6 +5731,24 @@ class LossLayer(LayerBase):
   But this loss will not be used as a loss by the updater.
   If you want to use it as a loss, you can use the :class:`AsIsLoss`,
   i.e. write ``"loss": "as_is"``.
+
+  Note that the loss options for the wrapped loss need to be provided via ``loss_opts_``,
+  and it does not apply any reduce function.
+
+  .. note::
+
+    The ``LossLayer`` might be deprecated in the future in favor of implementing the losses as actual layers.
+
+    If you want to define a loss inside the network, it is recommended to define it explicitly via an ``eval`` layer.
```

Comment:
No, this is not what I recommend. Or at least not the part about `eval`. This is actually what I recommend to avoid (`EvalLayer`). Maybe just remove the ("via an ``eval`` layer").

Reply:
But you did recommend exactly this:

> The equivalent layer, and what I suggest what you should do instead, is e.g.:
> 
> ```
> "mse_layer": {"class": "eval", "eval": "(source(0) - source(1)) ** 2", "from": ["output", "data:classes"]}
> ```

And it is impossible to define this without the help of an eval layer, or is it not?

Reply:
I very clearly wrote that I would avoid using the `EvalLayer`. This was just an example.

Maybe just remove the "via an eval layer" and this is ok.

> And it is impossible to define this without the help of an eval layer, or is it not?

Why? RETURNN should provide always all basic building blocks. If sth is missing, you should add it. But here we have all?

Reply:
I guess the `CombineLayer` is missing a "power" function, but yes that could easily be added. I will just remove the hint on "eval" layer...